frontend/src/components/molecules/ModalWindow/ModalWindow.styled.ts

```diff
@@ -0,0 +1,82 @@
+import Button from "components/atoms/Button";
+import styled, { keyframes } from "styled-components";
+
+const fadeInAnimation = keyframes`
+    from {
+        opacity: 0;
+    } to {
+        opacity: 1;
+    }
+`;
+
+export const StyledContainer = styled.div`
```

Comment:
Why we are using styled-components and tailwind together? We should decide to use one of them